soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -1,162 +1,178 @@
 /* Copyright (c) 2022 Intel Corporation
  * SPDX-License-Identifier: Apache-2.0
  */
-
  #include <zephyr/spinlock.h>
 
-#include <cavs_ipc.h>
+#include <intel_adsp_ipc.h>
+#if defined(CONFIG_SOC_SERIES_INTEL_ACE1X)
+#include <ace-ipc-regs.h>
+#include <ace_v1x-regs.h>
+#elif defined(CONFIG_INTEL_ADSP_CAVS)
 #include <cavs-ipc-regs.h>
+#endif
+#include <autoconf.h>
```

Comment:
why are you including this? This is already included by the build system and should not be included directly.